pytest.ini

```diff
@@ -5,6 +5,8 @@ markers =
     multiclass
     wandb_args
     flaky: tests marked as flaky need to be debugged
+    server_issues: tests that fail against the mock server with connection errors
```

Comment:
all the tests are under unit test so there is no server involved this marker seems wrong... could you please review it?